tests/integration/test_issue_comment.py

```diff
@@ -109,7 +109,7 @@ def test_issue_comment_propose_update_handler(
     )
     flexmock(
         GithubProject,
-        get_files=lambda ref, filter_regex: [],
+        get_files=lambda ref, filter_regex, recursive: [],
```

Comment:
didn't you mean this? :)

```suggestion
        get_files=lambda ref, filter_regex: [], recursive,
```

Reply:
I didn't, or am I missing something? O.o
`get_files` takes ref, regex **and** whether it is recursive search

Reply:
oh, I confused the syntax, yours is indeed correct